article/helpers.py

```diff
@@ -56,3 +72,15 @@ def retrieve_articles(self):
         )
         response.raise_for_status()
         return [article['article_uuid'] for article in response.json()]
+
+    def article_read_count(self, category):
+        if category not in structure.CATEGORIES_LOOKUP:
```

Comment:
the read count is not just for guidance categories but for all possible article lists